src/clients/host-client.js

```diff
@@ -279,20 +279,67 @@ class HostClient extends BaseEvernodeClient {
         return await this._getReputationInfoByAddress(this.reputationAcc.address);
     }
 
+    /**
+     * Prepare host reputation score to a common format for submission.
+     * @param {object} collectedScores 
+     * @returns Unified reputation score array.
+     */
+    async prepareHostReputationScores(collectedScores = {}) {
+        const moment = this.getMoment();
+        const myReputationInfo = await this.getReputationInfo();
+        const myOrderId = myReputationInfo.orderedId;
+
+        // Deciding universe.
+        const universeStartIndex = Math.floor(myOrderId / 64) * 64;
+        const universeEndIndex = Math.min(universeStartIndex + 64, 256);
+
+        const universeDetails = new Array(64);
+
+        for (let i = universeStartIndex; i < universeEndIndex; i++) {
+            const orderedIdMomentStateKey = StateHelpers.generateHostReputationOrderedIdStateKey(i, moment);
```

Comment:
There's a method called `getReputationInfoByOrderedId` in ReputationClient we can directly use it